src/OpenSage.Game/Logic/Player.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using OpenSage.Content;
+using OpenSage.Data.Ini;
+using OpenSage.Logic.Object;
+using OpenSage.Utilities.Extensions;
+
+namespace OpenSage.Logic
+{
+    // TODO: Abstract this out, or try to include all factions from every game?
+    public enum Side
```

Comment:
What benefit is there from this being an enum? I don't know if we'll need to have many/any `switch` statements based on this value.

Reply:
An enum felt like a better idea than an arbitrary string, but you're right that we're unlikely to benefit from the additional type safety.